public/static/docs/command-reference/config.md

```diff
@@ -114,18 +114,9 @@ for more details.) This section contains the following options:
   > option, properly transforming paths relative to the current working
   > directory into paths relative to the config file location.
 
-- `cache.protected` - make DVC-tracked files read-only. Possible values are
-  `true` or `false` (default). Run `dvc checkout` after changing the value of
-  this option for the change to go into effect.
-
-  Due to the way DVC handles linking between the data files in the cache and
```

Comment:
A bit sad to throw this out, but seems like it might be confusing to leave it as is from now on. Later we'll throw out `cache.protected` mentions for good.

Reply:
Not sad at all :) Best code == no code, right? )